src/Helper/Page.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace BusinessProfileRender\Page;
+
+/**
+ * Class Page
+ */
+class Page
+{
+    /**
+     * Initialize page functionality
+     */
+    public static function init()
+    {
+        add_action("admin_menu", [__CLASS__, "add_admin_page"]);
+        add_action("init", [__CLASS__, "add_action_wp_enqueue_styles"]);
+        add_action("init", [__CLASS__, "add_action_wp_enqueue_scripts"]);
+        add_action("init", [__CLASS__, "add_font_awesome"]);
+    }
+
+    public static function add_action_wp_enqueue_styles()
+    {
+        wp_enqueue_style(
+            "helperpage-style",
+            plugin_dir_url(__FILE__) . "../../assets/css/style.css"
+        );
+    }
+
+    public static function add_action_wp_enqueue_scripts()
+    {
+        wp_enqueue_script(
+            "helperpage-script",
+            plugin_dir_url(__FILE__) . "../../assets/js/helper-page.js"
+        );
+    }
+
+    public static function add_font_awesome()
+    {
+        wp_enqueue_style(
+            "font-awesome",
+            "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css"
+        );
+    }
+
+    /**
+     * Add admin page under Settings menu
+     */
+    public static function add_admin_page()
+    {
+        add_options_page(
+            "Business Profile Render", // Page title
+            "Business Profile Render", // Menu title
+            "manage_options", // Capability required to access the page
+            "page-settings", // Menu slug
+            [__CLASS__, "render_admin_page"] // Callback function to render the page
+        );
+    }
+
+    public static function render_admin_page()
+    {
+        // Retrieve serialized data from the database
+        global $wpdb;
```

Comment:
Use the `get_option()` function instead of using SQL quires [Ref](https://developer.wordpress.org/reference/functions/get_option/)